From c91491c08bd9449deddf7b3d507778201f8900c7 Mon Sep 17 00:00:00 2001
From: Piotr Gorski <lucjan.lucjanov@gmail.com>
Date: Sat, 29 Oct 2022 00:45:31 +0200
Subject: [PATCH 08/14] maple-tree: replace mas_find with mas_find_rev

Fix ERROR: modpost: "mas_find_rev" [lib/test_maple_tree.ko] undefined!

Signed-off-by: Piotr Gorski <lucjan.lucjanov@gmail.com>
---
 lib/maple_tree.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/maple_tree.c b/lib/maple_tree.c
index e33d4c2df..53f2fb84d 100644
--- a/lib/maple_tree.c
+++ b/lib/maple_tree.c
@@ -6064,7 +6064,7 @@ void *mas_find_rev(struct ma_state *mas, unsigned long min)
 	/* Retries on dead nodes handled by mas_prev_entry */
 	return mas_prev_entry(mas, min);
 }
-EXPORT_SYMBOL_GPL(mas_find);
+EXPORT_SYMBOL_GPL(mas_find_rev);
 
 /**
  * mas_erase() - Find the range in which index resides and erase the entire
-- 
2.38.1.385.g3b08839926

